roboprop_client/export_model.py

```diff
@@ -42,7 +42,7 @@ def generate_sdf():
             sdf = ElementTree.Element("sdf", attrib={"version": sdf_version})
             model = ElementTree.SubElement(sdf, "model", attrib={"name": model_name})
             static_xml = ElementTree.SubElement(model, "static")
-            static_xml.text = str(True)
+            static_xml.text = str(False)
```

Comment:
is it expected to have this commit in this PR?

Reply:
Thank you for spotting this. I think a poor / late rebase on my part has made this try to switch it back to True. Should be False